x-pack/plugins/aiops/public/components/log_rate_analysis/log_rate_analysis_page.tsx

```diff
@@ -324,6 +324,7 @@ export const LogRateAnalysisPage: FC<LogRateAnalysisPageProps> = ({
               <observabilityAIAssistant.ObservabilityAIAssistantContextualInsight
                 title={logRateAnalysisTitle}
                 messages={messages}
+                scope="observability"
```

Comment:
can we put this in the ObservabilityAIAssistantContextualInsight component itself?

Reply:
Done in https://github.com/elastic/kibana/pull/192611/commits/265fdac034ade57dee1a3f349f667e12f74e637c, now grabbing it from service instead of using as component input.